### DISCUSSION TITLE
Ad for a script to create activity heatmap from GarminDB

### DISCUSSION THREAD

**Discussion by j-paulus** [tcgoetz/GarminDB#92] [Ideas]
Finally having a full year activity data stored, I wanted to create a visualization of the lockdown walks and created a small Python script for it: [https://github.com/j-paulus/grrrmin_heatmap](https://github.com/j-paulus/grrrmin_heatmap) 

It uses the database from GarminDB in order to avoid parsing the .tcx/.fit/.gpx files self.

The features are for what I wanted (e.g., no interactive slippy map, but a static image), but maybe someone else might find it entertaining, too.

Thank you for your project!

**Comment by tcgoetz** [comment]
How about I add a section to the wiki for external/related projects?

**Reply by j-paulus** [reply]
That could be a good solution. I don't know how many projects are out there, but given the easy access to the data through GarminDB, more could come up.

**Reply by tcgoetz** [reply]
https://github.com/tcgoetz/GarminDB/wiki/Related-Projects
